### ISSUE TITLE
Link Dumps 2019

### ISSUE THREAD

**Issue by almereyda** [valueflows/valueflows#422]
Keeping up with finitely scrollable GitHub issue pages, here comes this year's slice of the infamous link dump series. #30 #91 #172 #269 #312

Starting with an article that reflects upon *intents* in online communities, in other words intentional communities, I'd like to direct your attention at

* [Thoughts from the Elm community on Intentional Communication - community - Discourse Meta](https://meta.discourse.org/t/thoughts-from-the-elm-community-on-intentional-communication/99332/6?u=almereyda) below and above.

How does this coalesce with our intuitions around *conversations for action*?

**Comment by bhaugen** [comment]
> Keeping up with finitely scrollable GitHub issue pages, here comes this year's slice of the infamous link dump series. #30 #91 #172 #269 #312

@almereyda thanks for rolling it over to the new year!

> How does this coalesce with our intuitions around _conversations for action_?

Very interesting thread, but I think not the same as CfA. CfA is a formalized protocol for coming to agreement (or explicitly failing to do so), and then following thru to performing and evaluating the agreed-upon actions. But it is only formalized for two agents. Doesn't work for more than two. 

Would be interesting to try to formalize such a protocol between many people. Roberts Rules of Order might be a starting point. But might be too formal...

**Comment by fosterlynn** [comment]
>Keeping up with finitely scrollable GitHub issue pages, here comes this year's slice of the infamous link dump series. #30 #91 #172 #269 #312

Yes, thanks, we count on you to do it! :sweat_smile:  Wow, first one in 2015, and here we are....
